regolith/helpers/a_projectumhelper.py

```diff
@@ -366,3 +371,30 @@ def insert_checklists(self, pdoc, now):
             "status": "finished"
         }})
         return pdoc
+
+    # create Google Doc page for prum in Google Drive 'bg-projects'
+    def gdoc_generator(self):
+        # To use prum name for generated document name
+        rc = self.rc
+
+        # Authenticate user
+        store = oauth2client.file.Storage('credentials.json')
```

Comment:
I don't remember what is assumed by oauth2client, but make sure it is looking in a universal place in the user's account hierarchy for this file.